src/sources/soundsourceffmpeg.cpp

```diff
@@ -933,8 +935,7 @@ bool SoundSourceFFmpeg::adjustCurrentPosition(SINT startIndex) {
     // Need to seek to a new position before continue reading. For
     // sample accurate decoding the actual seek position must be
     // placed BEFORE the position where reading continues.
-    // At the beginning of the stream, this is a negative position.
-    auto seekIndex = startIndex - m_seekPrerollFrameCount;
+    auto seekIndex = std::max(startIndex - m_seekPrerollFrameCount, (SINT)0);
```

Comment:
Can we replace `auto` with `SINT` and then use `0` without typecast?

Also, it may make sense to add a comment that `av_seek_frame` does not work with negative frame indices.